nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -1907,11 +1907,15 @@ that is specified.
     Rather, it is made available for extension points, such as Hadoop-based Processors, to use. At this time, only a single krb5 file is allowed to
```

Comment:
@mcgilman hard to see in the diff but I'm not sure that the statement shown in the line before is valid anymore?

"Note that this property is not used to authenticate NiFi users.  Rather, it is made available for extension points..." 

Looking at the KerboersServiceFactoryBean changes and the AccessResource changes it seems that now that file is involved in authentication. Is that a correct statement? If so I'm thinking the notes on the krb5.conf would need to be updated.